scripts/uploads/merge_and_uncode_rca_uploads.R

```diff
@@ -535,11 +508,13 @@ save_to_synapse <- function(path, parent_id, file_name = NA, prov_name = NA, pro
                     description = prov_desc,
                     used = prov_used,
                     executed = prov_exec)
-    file <- synStore(file, activity = act)
+    file_id <- synStore(file, activity = act)$properties$id
   } else {
-    file <- synStore(file)
+    file_id <- synStore(file)$properties$id
   }
-  
+  file_no_comment <- synGet(file_id, downloadFile=FALSE)
```

Comment:
As my understanding, this would generate a new version of the file with the comment. I suggest add the comment before store the file. It could be something like: 
```
file$properties$versionComment <- comment
file_id <- synStore(file, activity = act)$properties$id
```

Reply:
Agreed, I'm going to just go back to the original method (as placeholder)